.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
 - docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 - docker push docker.io/whoop/chat-ops:latest
 script:
-- docker version
+  - npm test
```

Comment:
This change is already present on the `master` branch. You may need to rebase this branch.